packages/slice-machine/src/modules/screenshots/sagas.ts

```diff
@@ -30,9 +30,14 @@ export function* generateSliceScreenshotSaga({
       screenDimensions,
     })) as SagaReturnType<typeof generateSliceScreenshotApiClient>;
 
+    // If screenshot is null, then no screenshots were taken
+    if (!response.data.screenshot) {
+      throw Error();
```

Comment:
Good, just send a unique error with it please, so we can trace it back if ever one day it happens.